README.md

```diff
@@ -442,7 +439,7 @@ body:
 {
   "sourceData": "12345678",
```

Comment:
Update to use underscores, source_event_data

Reply:
Done